plugins/portkey/gibberish.ts

```diff
@@ -10,7 +10,8 @@ import { PORTKEY_ENDPOINTS, fetchPortkey } from './globals';
 export const handler: PluginHandler = async (
   context: PluginContext,
   parameters: PluginParameters,
-  eventType: HookEventType
+  eventType: HookEventType,
+  options
```

Comment:
Can't we add `ts` support for this as well?!

Reply:
The handler already has `PluginHandler` type. So it will automatically infer options type from PluginHandler. Felt redundant to add it explicitly. Did not want to remove it from existing args in this PR.

Reply:
<img width="496" alt="Screenshot 2024-10-11 at 19 42 59" src="https://github.com/user-attachments/assets/c9e3f289-5dee-4f7a-902b-804f450cb793">